e2e/items/item-fetching.spec.ts

```diff
@@ -0,0 +1,67 @@
+import { test, expect } from '@playwright/test';
+import { initAsUsualUser } from '../helpers/e2e_auth';
+import { apiUrl } from 'e2e/helpers/e2e_http';
+
+/**
+ * Check that the basic item fetching (loading, breadcrumbs, results, ... work as expected)
+ */
+
+test('visit an already-started chapter', async ({ page }) => {
+  await initAsUsualUser(page);
+  await page.goto('/a/7523720120450464843;p=7528142386663912287;a=0');
+
+  await test.step('check breadcrumbs', async () => {
+    await expect.soft(page.locator('.breadcrumb-list li:nth-child(1)')).toContainText('Algorea Testing Content for devs');
+    await expect.soft(page.locator('.breadcrumb-list li:nth-child(3)')).toContainText('Tasks Showcase');
+  });
+
+  await test.step('check content', async () => {
+    await expect.soft(page.locator('h1.item-title')).toContainText('Tasks Showcase');
+    await expect(page.locator('.right-container').getByText('Blockly Basic Task')).toBeVisible();
```

Comment:
I would suggest to use testid in future

Reply:
Ok, I agree. For, the right container, I feel it is still somehow valid. And it is still use at other places... so maybe a task maybe to fix all locators across the tests.